stats/stats_test.go

```diff
@@ -29,120 +29,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestMetricHumanizeValue(t *testing.T) {
```

Comment:
These were very important tests IMO that we lose in the conversion to JS. We should probably add some for the JS implementation.

Reply:
I replicated (and IIRC even expanded a bit) these tests in https://github.com/k6io/jslib.k6.io/blob/master/tests/summary.js when I was writing the JS summary generation code.

Reply:
Ah, neat, I forgot about those. I suppose it's fine then, though running them as part of the k6 tests would give us more confidence, as they're run more frequently and use the latest k6 version. But this goes back to the point of eventually deprecating the jslib version, which we can consider later.

Reply:
In k6 we have somewhat higher-level tests in https://github.com/k6io/k6/blob/master/js/summary_test.go - they are like integration tests for the JS summary code, so I'd rather expand those than to write more low-level ones.